shell/ci/release/docker.sh

```diff
@@ -142,10 +142,16 @@ build_and_push_image() {
     fi
   fi
 
+  # Build image with registry prefix for devenv
+  if [[ -n $DEVENV_DOCKER_BUILD ]]; then
+    info "Building Docker Image for devenv"
+    image="gcr.io/outreach-docker/$image"
   # Build a quick native image and load it into docker cache for security scanning
   # Scan reports for release images are also uploaded to OpsLevel
   # (test image reports only available on PR runs as artifacts).
-  info "Building Docker Image (for scanning)"
+  else
+    info "Building Docker Image (for scanning)"
```

Comment:
Nit: I would move the comments into the `else` block, right before the `info` log and at the same indentation as `info` (GitHub won't let me make a suggestion for that).

Reply:
++, I just pushed an update for fixing this. You could be editing in the gap.

Reply:
Sorry, this is what I meant (manual suggestion block)

```suggestion
  else
    # Build a quick native image and load it into docker cache for security scanning
    # Scan reports for release images are also uploaded to OpsLevel
    # (test image reports only available on PR runs as artifacts).
    info "Building Docker Image (for scanning)"
```

Reply:
ahh, good catch, thanks. Github didn't let me commit from the here directly, I manually modified it as suggestion.